### PR TITLE
Release v0.06

### PR DESCRIPTION
**Pull request by jessb0t**
@georgebuzzell Please review and merge our sixth release. 🤓 This contains:
* Modification of NA value to "NA" (previously "N/A").
* Support for double digit event numbers and double letter instrument versioning
* Modification of perc calculations for instruments with products to provide transparent percentage across all questions (not across products)
* Addition of flat to "hide" certain scores from the output (e.g., the interim products for AQ10).

Thank you! 🙏

### CODE REVIEW

**Review by georgebuzzell — APPROVED** [review]

**Comment by georgebuzzell** [comment]
@jessb0t looks good. Just a super minor comment.

Also, two minor questions: 

What is an example of a survey that involves a product calculation?

Can you please provide slightly more context here:
"Modification of perc calculations for instruments with products to provide transparent percentage across all questions (not across products)

**Comment by jessb0t** [comment]
@georgebuzzell Thanks!  The AQ10 is an example of a survey that involves product calculation.  Two products "scrdAgr" and "scrdDis" must first be calculated, and then the actual questionnaire score (aq10_scrdTotal) is the sum of those.  Before this release, the percentage calculation for the total score (aq10_percTotal) was looking at the products, not the sub-items that comprise the products.  So, for example, if a participant was missing the answer to one question on the AQ10, you would see aq10_percTotal = 0.5 (when, in fact, it should be 0.9) because the script was unable to score one of the product and was using these products to calculate percentage complete.

image

**Comment by jessb0t** [comment]
@georgebuzzell I am doing some administrative cleanup on the REDCap uploads following our study setup meetings so that the .zip files will automatically have the survey title and instructions in them.  Once that is complete, I will merge to `main`!

File: README.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ The following software iterations are planned for development. Each iteration is
 * Addition of unscored surveys: demo_c, postTaskD, thq_b.
 * Addition of initial data dictionaries.
 
+### 0.06
+
+* Modification of NA value to "NA" (previously "N/A").
+* Support for double digit event numbers and double letter instrument versioning
+* Modification of perc calculations for instruments with products to provide transparent percentage across all questions (not across products)
+* Addition of flat to "hide" certain scores from the output (e.g., the interim products for AQ10).
```
[the remark this answers is not in the document]
Comment (jessb0t) [reply README.md]: Corrected!

**Pull request merged**